src/UI/UIUtils.java

```diff
@@ -335,15 +335,18 @@ public static COSpriteSpec fromDisk(String col, String fac)
   public static final COSpriteSpec LA = COSpriteSpec.fromDisk("Lazurian"      , "Army");
   public static final COSpriteSpec IDS = COSpriteSpec.fromDisk("IDS"          , "Army");
 
+  public static COSpriteSpec[] CANON_FACTIONS;
+  public static SourceGames[] GAME_RESOLUTION_ORDER;
+
   public enum SourceGames
```

Comment:
Is it just me or is it strange to have the enum type plural?

Like, if I wanted to create an enum for the type of vehicle someone had, I wouldn't name it `VehicleTypes`, and then enumerate trains, planes, an automobiles, I'd call it `VehicleType`.

Reply:
I didn't know that was a rule.
My hands just type words and then there's code.